hack/make/go.mk

```diff
@@ -29,7 +29,7 @@ go/test-all:
 go/run: export RUN_LOCAL=true
 go/run: export POD_NAMESPACE=dynatrace
 go/run: manifests/kubernetes manifests/openshift go/fmt go/vet
-	go run ./src/cmd/operator/
+	go run ./src/cmd/ operator
```

Comment:
🤔

Reply:
Looks like a typo but is actually correct. - We moved our main.go to ./src/cmd and added some subcommands. The one we need here is `operator`.